### DISCUSSION TITLE
Method For Delineation Between Sites

### DISCUSSION THREAD

**Discussion by mr1716** [mattieserver/netbox-topology-views#492] [Ideas]
Pardon me if this is already configurable, but new to the plugin use. Without the use of maps, when selecting multiple sites or regions, there currently isnt a way to visually tell which devices are at which sites. It would be extremely useful to have a visual way to show which devices are at each sites. It could be a box surrounding the devices at each site, which each box labelled with the applicable site name.

**Comment by dreng** [comment]
Hi,

this has been implemented in the latest release (v3.9.1). See #462

**Reply by mr1716** [reply]
@dreng thanks for the information, as I will have to check it out. Also having a way to automatically do that would be a future improvement. Are there to have the boxes automatically be drawn instead of having to be drawn or added manually? (Not saying the ability to manually draw should be removed, but rather asking if the ability to auto draw would be implemented in a future version)
